src/TouristInfoCards.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
```

Comment:
This looks great on your page.

Notice though that you have very similar code for each of the three cards since they all have the same structure (all the HTML elements are the same, it's just the content that changes). When you see code repeated, that's usually a good sign that you can simplify it by creating a separate component.

Why don't you try creating a separate <Card> React component and store the city data in an array that you can then iterator over to create a new card component for each city. Similar to [the example that you did in class](https://codesandbox.io/s/interpolation-with-map-in-jsx-7mw0mw3qx0?file=/src/MentorsList.js).